src/v0/sources/shopify/util.js

```diff
@@ -37,7 +37,12 @@ const getShopifyTopic = (event) => {
   }
   return topic[0];
 };
-
+const getHashLineItems = (cart) => {
+  if (cart && cart?.line_items && cart.line_items.length > 0) {
+    return sha256(JSON.stringify(cart.line_items));
+  }
+  return "0";
```

Comment:
should we use - "EMPTY" more descriptive right?

Reply:
Yes. This 0 was result of design discussion. Any how `EMPTY` would make much more sense. Changing it

Reply:
Done